template/src/index.ts.ejs

```diff
@@ -38,7 +40,9 @@ export function <%- ex.name %>(): number {
 
   <% if (ex.output) { -%>
     <% if (isJsonEncoded(ex.output)) { -%>
-      <% if (isPrimitive(ex.output)) { -%>
+      <% if (isBuffer(ex.output)) { -%>
+        Host.outputString(JSON.stringify(Host.arrayBufferToBase64(output)))
```

Comment:
Ditto here – we can unwrap views:
```suggestion
        Host.outputString(JSON.stringify(Host.arrayBufferToBase64(ArrayBuffer.isView(output) ? output.buffer : output)))
```

Reply:
(Actually, I take this back – the view has a `.byteOffset` that we'd need to respect by slicing the buffer!)